frames/src/main/java/com/checkout/frames/style/component/default/DefaultLightStyle.kt

```diff
@@ -142,7 +145,7 @@ public object DefaultLightStyle {
         inputFieldStyle = inputFieldStyle(withLeadingIcon).apply {
             placeholderText = placeholderResourceText
             placeholderTextId = placeholderResourceTextId
-        },
+        }.copy(keyboardOptions = keyboardOptions),
```

Comment:
It would be nice to update arguments for `public fun inputFieldStyle(withLeadingIcon` instead of copy.